src/components/Breadcrumb/Breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import Breadcrumb from './Breadcrumb';
+import dedent from 'dedent';
+
+const Breadcrumbs = styled.div`
+  &>*:not(:last-child)::after {
```

Comment:
Since we're already being specific about child component type, we can use the more efficient class name selector `${Breadcrumb}` here instead of `*`.